s3/templates/cloudtrail.json.tpl

```diff
@@ -0,0 +1,28 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "AWSCloudTrailAclCheck",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "cloudtrail.amazonaws.com"
+      },
+      "Action": "s3:GetBucketAcl",
+      "Resource": "arn:aws:s3:::${bucket_name}"
+    },
+    {
+      "Sid": "AWSCloudTrailWrite",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "cloudtrail.amazonaws.com"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::${bucket_name}/*",
+      "Condition": {
+        "StringEquals": {
+          "s3:x-amz-acl": "bucket-owner-full-control"
```

Comment:
I looked up what this condition does, and now I'm a bit confused. The example in https://docs.aws.amazon.com/AmazonS3/latest/dev/amazon-s3-policy-keys.html#AvailableKeys-iamV2 uses `StringNotEquals` rather than `StringEquals`, even though it sounds like it's trying to achieve the same thing:

```
"Condition": {
	"StringNotEquals": {
		"s3:x-amz-acl": "bucket-owner-full-control"
	}
```

Your version makes more sense to me, so maybe it's a copy-paste error in the AWS docs? Or is that an example from a `Deny` policy?

Reply:
I copied the policy from https://www.terraform.io/docs/providers/aws/r/cloudtrail.html and it works correctly in the TDR Management account

On the link you sent there are several examples, most with Allow policies using StringEquals. StringNotEquals appears in a Deny policy.